### PR TITLE
Chip documentation cleanup

### PR DESCRIPTION
**Pull request by rey-wright**
## Description
- removed size="lg" prop from InlineBlockList chip examples (size doesn't exist)
- moved `noop` to general utility folder for reuse (I believe we're using this elsewhere but not going to search for that now)
- reorganized proptype definitions in order of importance (which will reflect in Storybook

## Screenshots
N/A

## Checklist
- [x] Docs have been rebuilt (`yarn build:full`)
- [x] All unit tests pass
- [x] Snapshots have been updated
- [x] No lint errors or warnings have been introduced
- [x] **[Version bumped](https://github.com/cbinsights/form-design-system#updating-version-number) if appropriate**

### CODE REVIEW

File: src/components/util/noop.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const noop = () => {};
```
Comment (akdetrick) [inline src/components/util/noop.js]: `() => {}` is es6 convention for a noop. Not sure we need a utility for this, but I'm not against it either

Comment (rey-wright) [reply src/components/util/noop.js]: hmmm so I agree in a sense... but usually when I do noop's, I do them inline. The fact that you broke it out into a variable, was a signal to me that we'd have multiple noop variables in different files, and I'd just break it out into a utility. I guess my follow up question would be... do we need variables for this?

Comment (akdetrick) [reply src/components/util/noop.js]: probably not - if I stored a noop in a var, I probably shouldn't have

Comment (rey-wright) [reply src/components/util/noop.js]: yeah. You have noops defined in both this story, and the popover component. I'll wipe them away (if that's what you want)

Comment (rey-wright) [reply src/components/util/noop.js]: actually... the subsequent one in Popover may have been Nico.

**Pull request merged**